app/Nova/ActionGuards/UitIdv1/ActivateUitIdv1ClientGuard.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova\ActionGuards\UitIdv1;
+
+use App\Nova\ActionGuards\ActionGuard;
+use App\UiTiDv1\UiTiDv1ClusterSDK;
+use App\UiTiDv1\UiTiDv1Consumer;
+use App\UiTiDv1\UiTiDv1ConsumerStatus;
+
+final class ActivateUitIdv1ClientGuard implements ActionGuard
```

Comment:
We could start using `readonly classes` from PHP 8.2, see https://stitcher.io/blog/readonly-classes-in-php-82

Maybe a good enhancement to pick up in a follow-up pull request?

Reply:
Changed it for my ticket, but there are many classes left to be a candidate for this change.